kubernetes/settings

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
```

Comment:
so this is a bash script, it's better to name the file as xxxx.sh?

Reply:
`settings` is consistent with both the instructions in the `gen` repo and the other client repos...